src/App.tsx

```diff
@@ -1,9 +1,67 @@
+import { useCallback, useEffect, useState } from 'react';
 import './App.scss';
+import {
+  Link, Redirect, Route, Switch,
+} from 'react-router-dom';
+import { PeopleTable } from './components/PeopleTable';
+import { getPeople } from './api';
+import { Person } from './PersonType';
 
-const App = () => (
-  <div className="App">
-    <h1>People table</h1>
-  </div>
-);
+const App = () => {
+  const [people, setPeople] = useState<Person[] | null>(null);
+  const [textError, setTextError] = useState<string>('');
+
+  const getData = useCallback(async () => {
+    try {
+      const gottenPeople = await getPeople();
+
+      setPeople(gottenPeople);
+      setTextError('');
+    } catch {
```

Comment:
I think it meant to create and call this in PeopleTable

Reply:
I used it here especially to train using props, but if mentor says I'll do it.

Reply:
It can be here as for me